Lib/ldif.py

```diff
@@ -209,7 +209,7 @@ def unparse(self,dn,record):
 def CreateLDIF(dn,record,base64_attrs=None,cols=76):
   """
   Create LDIF single formatted record including trailing empty line.
-  This is a compatibility function. Use is deprecated!
+  This is a compatibility function.
```

Comment:
``warnings.warn('CreateLDIF is deprecated, use LDIFWriter.unparse() instead.', category=DeprecationWarning, stacklevel=2)``